src/Resources/views/CRUD/list_outer_rows_mosaic.html.twig

```diff
@@ -62,16 +60,29 @@ This template can be customized to match your needs. You should only extends the
                             {% endif %}
 
                             {% block sonata_mosaic_description %}
-                                {% if admin.hasAccess('edit', object) and admin.hasRoute('edit') %}
-                                    <a class="mosaic-inner-link" href="{{ admin.generateUrl('edit', {'id' : object|sonata_urlsafeid(admin) }) }}">{{ meta.title|truncate(40) }}</a>
-                                {% elseif admin.hasAccess('show', object) and admin.hasRoute('show') %}
-                                    <a class="mosaic-inner-link" href="{{ admin.generateUrl('show', {'id' : object|sonata_urlsafeid(admin) }) }}">{{ meta.title|truncate(40) }}</a>
-                                {% else %}
-                                    {{ meta.title|truncate(40) }}
-                                {% endif %}
+                                {{ meta.title|truncate(40) }}
                             {% endblock %}
                         </div>
                     </div>
+                {% endset %}
+                <div class="col-xs-6 col-sm-3 mosaic-box sonata-ba-list-field-batch sonata-ba-list-field"
+                     objectId="{{ admin.id(object) }}"
+                >
```

Comment:
We usually don't put that sign on its own line

Reply:
should I replace `class` and `objectId` position? I mean it was a long line I had to break it somewhere :smile:

Reply:
`class` and `objectId` are perfect, just do this: objectId="{{ admin.id(object) }}">

Reply:
got it, did not understand what you were talking about :sweat_smile: